types/externs.d.ts

```diff
@@ -202,10 +196,6 @@ declare global {
       group: string;
     }
 
-    export interface LighthouseError extends Error {
```

Comment:
might have been nice to just export our class as this type so we could easily reference it as `LH.LighthouseError` still but I don't feel strongly

Reply:
yeah, I'd like to add this to our public types (when those actually exist :)